source/includes/_tipser-elements.md

```diff
@@ -190,6 +190,9 @@ To activate the Cart, you need to dedicate an element on your page to host a sho
 The cart icon can be placed anywhere on your website. If you want to keep it visible at all times, please follow the [instructions](#cart).
 
 
+## Versioning
+  
+Tipser Elements is following <a href="https://semver.org/" target="_blank">Semantic Versioning</a>. This means that an increase in the major number in our version indicates potential <b>breaking changes</b>. Please be aware of that! For the react version, it is recommended to auto-update to the latest version with the same major number (see the caret (^) character in `package.json` file described <a href="https://stackoverflow.com/a/22345808" target="_blank">here</a>). Be aware, that tipser-elements (non-react) is always updating to the newest version published, even if the increased major version is introduced.
```

Comment:
Can we wrap this part: "Be aware, that tipser-elements (non-react) is always updating to the newest version published, even if the increased major version is introduced." in a danger box, like below?

![image](https://user-images.githubusercontent.com/1180737/76779463-adbaf200-67ab-11ea-8138-6501fec669d3.png)